### PR TITLE
fix bug 18/10

### PR DESCRIPTION
**Pull request by danhloimta**

### CODE REVIEW

File: app/Repositories/Eloquents/BookEloquentRepository.php [hunk 1]
```diff
@@ -255,4 +255,22 @@ public function search($attribute, $data, $with = [], $dataSelect = ['*'])
...
+    {
+        $book = $this->model()->findOrFail($id);
+
+        $star = 0;
+        foreach ($data as $review) {
+            $star += $review->star;
+        }
+
+        if ($data->count() > 0) {
+            $Avgstar['avg_star'] = $star/count($data);
```
Comment (framgiaciv3) [inline app/Repositories/Eloquents/BookEloquentRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookEloquentRepository.php</br>Variable "Avgstar" is not in valid camel caps format

File: app/Repositories/Eloquents/BookEloquentRepository.php [hunk 2]
```diff
@@ -255,4 +255,22 @@ public function search($attribute, $data, $with = [], $dataSelect = ['*'])
...
+        } else {
+            $Avgstar['avg_star'] = $star;
```
Comment (framgiaciv3) [inline app/Repositories/Eloquents/BookEloquentRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookEloquentRepository.php</br>Variable "Avgstar" is not in valid camel caps format

File: app/Repositories/Eloquents/BookEloquentRepository.php [hunk 3]
```diff
@@ -255,4 +255,22 @@ public function search($attribute, $data, $with = [], $dataSelect = ['*'])
...
+        }
+
+        return $book->update($Avgstar);
```
Comment (framgiaciv3) [inline app/Repositories/Eloquents/BookEloquentRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookEloquentRepository.php</br>Variable "Avgstar" is not in valid camel caps format

File: app/Http/Controllers/User/ReviewBookController.php [hunk 4]
```diff
@@ -55,7 +55,11 @@ public function store(ReviewRequest $request, $slug)
         try {
             $id = (int)last(explode('-', $slug));
             $request->merge(['book_id' => $id]);
+
             $this->review->store($request->all());
+            $data = $this->review->find($id);
```
Comment (thanhhungchu95) [inline app/Http/Controllers/User/ReviewBookController.php:60]: không tìm thấy record thì ko xử lý à

File: app/Http/Controllers/User/ReviewBookController.php [hunk 5]
```diff
@@ -55,7 +55,11 @@ public function store(ReviewRequest $request, $slug)
...
+
+            $this->book->updateStar($data, $id);
```
Comment (thanhhungchu95) [inline app/Http/Controllers/User/ReviewBookController.php]: nếu $data null thì sao

File: app/Repositories/Eloquents/BookEloquentRepository.php [hunk 6]
```diff
@@ -255,4 +255,22 @@ public function search($attribute, $data, $with = [], $dataSelect = ['*'])
...
+    {
+        $book = $this->model()->findOrFail($id);
+
+        $star = 0;
+        foreach ($data as $review) {
+            $star += $review->star;
+        }
+
+        if ($data->count() > 0) {
+            $avgStar['avg_star'] = $star/count($data);
```
Comment (thanhhungchu95) [inline app/Repositories/Eloquents/BookEloquentRepository.php]: thêm 1 space trước và sau /

File: app/Repositories/Eloquents/BookEloquentRepository.php [hunk 7]
```diff
@@ -255,4 +255,22 @@ public function search($attribute, $data, $with = [], $dataSelect = ['*'])
             ->search($attribute, $data)
             ->get();
     }
+
+    public function updateStar($data, $id)
+    {
+        $book = $this->model()->findOrFail($id);
+
+        $star = 0;
+        foreach ($data as $review) {
```
Comment (thanhhungchu95) [inline app/Repositories/Eloquents/BookEloquentRepository.php]: không có check null này

File: resources/assets/user/css/custom.css [hunk 8]
```diff
@@ -463,3 +459,11 @@ ul.list-unstyled.social-list i {
     margin: 11px 0 2px 0;
     display: block;
 }
+
+.product-rating li i, .rating-summary i {
+    color: #F07C29;
```
Comment (thanhhungchu95) [inline resources/assets/user/css/custom.css]: viết dạng chữ thường #f07c29

**Comment by thanhhungchu95** [comment]
commented
còn lỗi ci kìa

**Pull request merged**